### PR TITLE
Underscore the Copy Statement if the Copybook Contains a Syntax Error

### PR DESCRIPTION
**Pull request by temanbrcom**

### CODE REVIEW

**Comment by temanbrcom** [comment]
@grianbrcom 
@asatklichov

File: server/src/main/java/com/broadcom/lsp/domain/common/model/Position.java [hunk 1]
```diff
@@ -19,17 +19,37 @@
...
+        && line == position.line
+        && charPositionInLine == position.charPositionInLine
+        && documentURI.equals(position.documentURI)
+        && Objects.equals(token, position.token);
+  }
+
+  @Override
+  public int hashCode() {
+    return Objects.hash(documentURI, startPosition, stopPosition, line, charPositionInLine, token);
+  }
```
Comment (temanbrcom) [inline server/src/main/java/com/broadcom/lsp/domain/common/model/Position.java:54]: To be changed in #486

**Review by asatklichov — APPROVED** [review]

File: server/src/test/java/com/ca/lsp/cobol/usecases/engine/parser/TestPreprocessorParser.java [hunk 2]
```diff
@@ -12,22 +12,6 @@
...
- *
- * SPDX-License-Identifier: EPL-2.0
- *
- * Contributors:
- *    Broadcom, Inc. - initial API and implementation
- *
- */
-
 package com.ca.lsp.cobol.usecases.engine.parser;
 // Generated from .\TestPreprocessor.g4 by ANTLR 4.7.2
```
Comment (sergiuilie) [inline server/src/test/java/com/ca/lsp/cobol/usecases/engine/parser/TestPreprocessorParser.java:16]: please rename the grammar name

**Review by sergiuilie — APPROVED** [review]

**Comment by temanbrcom** [comment]
Added recursive raising of errors to the top of the document hierarchy

**Review by zacanbrcom — APPROVED** [review]
🔧

**Pull request merged**